docs/en_US/deployment/on-gcp-gke.md

```diff
@@ -1,60 +1,160 @@
+## Overview
 
-## Create GKE Cluster
+This guide will walk you through the process of deploying EMQ X, an open-source MQTT broker, on Google Kubernetes Engine (GKE). By following these steps, you'll learn how to create and configure an EMQ X deployment on GKE.
 
-Log in Google Cloud GKE console and enter the page for creating GKE Cluster. For details: [Create GKE cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-an-autopilot-cluster)
+## Prerequisites
 
-## Access Kubernetes Cluster
++ A Google Kubernetes Engine (GKE) cluster, for more information, see [Creating an autopilot cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-an-autopilot-cluster)
++ MQTTX CLI, A user-friendly MQTT 5.0 command line tool, download it [here](https://mqttx.app/cli)
 
-For details: [kubeconfig](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
 
-## Configure Cert Manager
+## Deploy EMQX on GKE
 
-To install `cert-manager` read the official docs:
+### Deploying EMQX Operator
+
+**Deploying Cert Manager**
+
+To install `cert-manager`, consult the official documentation:
 
 - [GKE Autopilot](https://cert-manager.io/docs/installation/compatibility/#gke-autopilot)
 - [Private GKE Cluster](https://cert-manager.io/docs/installation/compatibility/#gke)
 
-Don't forget to install CRDs when running `helm` using `--set installCRDs=true`.
+Remember to install CRDs when running `helm` with the `--set installCRDs=true` flag.
 
-> More info at [cert-manager](https://cert-manager.io).
+> More information can be found at [cert-manager](https://cert-manager.io).
 
-## EMQX Operator
+To install `emqx-operator`, refer to the official [docs](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md)
 
-Deploy it as usual.
+## Check available storage class
 
-## Create EMQX Cluster
+```Shell
+kubectl get sc
+```
+We use `standard` as storage class in the guide
 
-[Operator installation reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md)
 
-After Operator is installed, deploy EMQX cluster in GKE as usual.
+### Deploying EMQX Cluster
 
-### Troubleshoot: Context deadline exceeded when calling "mutating.apps.emqx.io" webhook
+Below are the relevant configurations for EMQX Custom Resource. Choose the corresponding APIVersion based on the EMQX version you want to deploy. For specific compatibility relationships, See [EMQX Operator Compatibility](../README.md):
 
-Before deploying your EMQX CRD, you need to make sure yout VPC doesn't have any firewall rules that denies traffic to the EMQX webhook.
-Open port 443 to allow receiveing ingress traffic.
+:::: tabs type:card
+::: tab v2alpha1
 
-## Persistence
+```yaml
+apiVersion: apps.emqx.io/v2alpha1
+kind: EMQX
+metadata:
+  name: emqx
+spec:
+  image: "emqx:5.0"
+  coreTemplate:
+    spec:
+      volumeClaimTemplates:
+        storageClassName: standard
```

Comment:
The default storageCalssName in Google is `standard` ?

Reply:
yes